sycl/doc/design/Fuzzing.md

```diff
@@ -0,0 +1,104 @@
+# Fuzzing plan for intel/llvm
+
+The main product which is being developed at intel/llvm repo is a SYCL
+implementation. At high-level, it consists of two compiler and runtime and
+therefore this document will be divided into two major sections covering those
+components. Those components are essentially the only entry points through which
+a user can interact with our product.
+
+## SYCL Runtime
+
+SYCL runtime is a library which implements SYCL API, but besides that API it
+also has multiple configuration options which can be tweaked through environment
+variables and config file.
+
+### Fuzzing environment variables
+
+Following environment variables [documentation][sycl-rt-env-variables] every env
+variable should be fuzzed.
+
+The most interesting are ones which expect data in a certain format, like
```

Comment:
I would mention that we are still talking about env vars, given it is a new paragraph.

```suggestion
The most interesting of the environment variables are ones which expect data in a certain format, like
```

Reply:
Applied in db6aba9154a2989f998a7c77ac8e388160dd9b18